VisualRiders.PointOfSale.Project/Services/TaxesService.cs

```diff
@@ -0,0 +1,74 @@
+using AutoMapper;
+using VisualRiders.PointOfSale.Project.DTOs;
+using VisualRiders.PointOfSale.Project.Models;
+using VisualRiders.PointOfSale.Project.Repositories;
+
+namespace VisualRiders.PointOfSale.Project.Services;
+
+public class TaxesService
+{
+    private readonly TaxesRepository _repository;
+    private readonly IMapper _mapper;
+
+    public TaxesService(TaxesRepository repository, IMapper mapper)
+    {
+        _repository = repository;
+        _mapper = mapper;
+    }
+
+    public ReadTaxDto Create(CreateUpdateTaxDto dto)
+    {
+        var taxesEntity = _mapper.Map<Tax>(dto);
+
+        taxesEntity.BusinessEntityId = 1;
+        
+        _repository.Add(taxesEntity);
+        _repository.SaveChanges();
+
+        return _mapper.Map<ReadTaxDto>(taxesEntity);
+    }
+
+    public List<ReadTaxDto> GetAll()
+    {
+        return _repository.GetAll().Select((tax) => _mapper.Map<ReadTaxDto>(tax)).ToList();
```

Comment:
Would be cleaner if you just passed the method reference instead of using the lambda syntax